cmd/integration_test/cmd/run.go

```diff
@@ -62,6 +63,7 @@ func init() {
 	runE2ECmd.Flags().IntP(maxInstancesFlagName, "m", 1, "Run tests in parallel with multiple EC2 instances")
 	runE2ECmd.Flags().StringSlice(skipFlagName, nil, "List of tests to skip")
 	runE2ECmd.Flags().Bool(bundlesOverrideFlagName, false, "Flag to indicate if the tests should run with a bundles override")
+	runE2ECmd.Flags().StringSliceP(gates, "g", nil, "List of feature gates to activate")
```

Comment:
Do we need this feature? Can't we just enable it in code as part of the "setup" process? It simplifies the interface of the cli and it's easier to find in code where at which flags are being enabled

Reply:
I was initially intending to do it in `setup`. But I thought then anyone who adds tests for a feature gated feature would need to add something in setup.
Instead my idea was we could enable all the gates we want for e2e testing from a command flag?

Reply:
The problem with that is the feature gate will be enabled even for the tests that "don't need it"

Reply:
Actually I thought that might be helpful. Since it would ensure that all the tests are passing with the gate enabled.
So we know that the new feature doesn’t break any of the existing ones.

Reply:
As discussed, reverted changes in favor of enabling gates for corresponding tests.